claims.go

```diff
@@ -37,7 +37,7 @@ func (c StandardClaims) Valid() error {
 	// default value in Go, let's not fail the verification for them.
 	if c.VerifyExpiresAt(now, false) == false {
 		delta := time.Unix(now, 0).Sub(time.Unix(c.ExpiresAt, 0))
-		vErr.Inner = fmt.Errorf("token is expired by %v", delta)
+		vErr.Inner = fmt.Errorf("Token is expired by %v", delta)
```

Comment:
I am afraid of ill-advisable use of the error message as well. It seems like there should be a better way to send this information to the user rather than sending it via an error message.